redwood-protocol/src/commonMain/kotlin/app/cash/redwood/protocol/protocol.kt

```diff
@@ -37,7 +37,7 @@ public data class Event(
   val id: Id,
   /** Identifies which event occurred on the widget with [id]. */
   val tag: EventTag,
-  val value: JsonElement = JsonNull,
+  public val args: List<JsonElement> = emptyList(),
```

Comment:
I was flirting with making this not a `data class`, but I'll save that for #1043 
```suggestion
  val args: List<JsonElement> = emptyList(),
```